mobile-widgets/qmlmanager.cpp

```diff
@@ -1847,6 +1849,17 @@ void QMLManager::showDownloadPage(QString deviceString)
 	emit pluggedInDeviceNameChanged();
 }
 
+void QMLManager::setFilter(const QString filterText)
+{
+	// show that we are doing something, then do something in another thread in order not to block the UI
+	QMetaObject::invokeMethod(qmlWindow, "showBusy");
+	QFuture<void> future = QtConcurrent::run(QThreadPool::globalInstance(),
```

Comment:
`future` is never used - am I missing something?

Reply:
Good catch. I thought I'd use a FutureWatcher to know when this is done - but then ended up simply doing the call to hide in the lambda

(BTW: I can't believe that no one has called me out of using a lambda in my implementation - I was certain that I'd get some serious ribbing over this one...)